doc/whats_new/v0.21.rst

```diff
@@ -68,6 +68,15 @@ Support for Python 3.4 and below has been officially dropped.
   between 0 and 1.
   :issue:`13086` by :user:`Scott Cole <srcole>`.
 
+- |Enhancement| Made `calibration.CalibratedClassifierCV` inherit from
```

Comment:
This change isn't user facing. No need to litter the change log with such things IMO.

Reply:
I agree, done